src/progly/query.cc

```diff
@@ -404,7 +418,96 @@ void worker_exec_runstats_op(librados::IoCtx *ioctx, stats_op op)
   ioctx->close();
 }
 
+void worker_init_lock_obj_op(librados::IoCtx *ioctx, inbl_lockobj_info op)
+{
+    std::string oid = op.table_group;
+
+    std::cout << "We spawned one thread here" << std::endl;
+    ceph::bufferlist inbl, outbl;
+    ::encode(op, inbl);
+    int ret = ioctx->exec(oid, "tabular", "inittable_group_obj_query_op",
+                          inbl, outbl);
+    checkret(ret, 0);
+    //print_data(&outbl);
+    ioctx->close();
+
+
+
+}
+
+void worker_create_lock_obj_op(librados::IoCtx *ioctx, inbl_lockobj_info op)
+{
+    std::string oid = op.table_group;
+
+    std::cout << "We spawned one thread here" << std::endl;
+    ceph::bufferlist inbl, outbl;
+    ::encode(op, inbl);
+    int ret = ioctx->exec(oid, "tabular", "create_lock_obj_query_op",
+                          inbl, outbl);
+    checkret(ret, 0);
+    //print_data(&outbl);
+    ioctx->close();
+
+
+
+}
+void worker_free_lock_obj_op(librados::IoCtx *ioctx, inbl_lockobj_info op)
+{
+
+    std::cout << "We spawned one thread here:" << op.table_name << std::endl;
+    std::string oid = op.table_group;
+    ceph::bufferlist inbl, outbl;
+    ::encode(op, inbl);
+    int ret = ioctx->exec(oid, "tabular", "free_lock_obj_query_op",
+                          inbl, outbl);
+    std::cout << "Returned value is:" << ret;
```

Comment:
i think we do not need to print this here, since heckret will throw and assert if error below.